src/loader/AMASSLoader.py

```diff
@@ -0,0 +1,320 @@
+import glob
+import json
+import os
+import random
+from datetime import datetime
+import numpy as np
+import shutil
+
+import mathutils
+
+from src.loader.LoaderInterface import LoaderInterface
+from src.utility.Utility import Utility
+from src.utility.LabelIdMapping import LabelIdMapping
+
+import torch
```

Comment:
torch is not listed in here: `"git+https://github.com/abahnasy/smplx", "git+https://github.com/abahnasy/human_body_prior"` 

So, I assume it is a dependency in these projects?

Reply:
Yes, It is included as a dependency for both libraries. In essence, I had two choices to incorporate the dependencies for this module, either to copy-paste the code needed from authors' libraries into a utility file inside BlenderProc or to install the authors' libraries themselves. I preferred to go with the second option to avoid any copyright issues. I just forked them and deleted dependencies that are not needed for BlenderProc, keeping what is necessary and also to avoid any bugs that may arise if the authors update the libraries later.

Reply:
I agree that makes the most sense.